### DISCUSSION TITLE
Google Play Protect triggered when installing Libre Tube 0.22.1

### DISCUSSION THREAD

**Discussion by dalaeance** [libre-tube/LibreTube#5790] [General]
### Steps to reproduce

1. Initiate install of Libre Tube 0.22.1 (in my case, via Neo Store)

### Expected behavior

No warnings should pop up?

### Actual behavior

Warning pops up, with the option to proceed anyway (screenshot included).

### LibreTube version

0.22.1

### Android version

Android 14

### Other details

![signal-2024-03-21-15-39-36-561](https://github.com/libre-tube/LibreTube/assets/30301418/b264c3c4-0a22-4c36-a64e-fe66eb21a909)

Really just sharing in case it's noteworthy. It's never happened to me before, so I came here to see if anyone else had come across this & reported it.

Would love a little discussion about what's new in this version that might have triggered it. Especially for the benefit of anyone who may experience this and come looking.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by ScriptSlayer0** [comment]
It's normal, only click on "install anyway".

**Comment by dalaeance** [comment]
I figured it's normal, but some insight into why it happens would still be nice. I'm not really concerned, given it's coming from a trusted source.

Also, realizing there's a Discussions section. Some projects don't have it, so I completely forgot to check.

**Comment by ScriptSlayer0** [comment]
It is mainly because Google (Android) detects that it is an application outside the Google play store, and pops up that dialogue to warn that it can be dangerous (only because it is outside the shop).
It wouldn't be bad for new users who see that message to put in the README, or somewhere saying that this behaviour is normal and not to worry.
